app/controllers/submit.js

```diff
@@ -75,87 +106,188 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
+    }),
+    // Does node have a saved description?
+    savedAbstract: Ember.computed('node.description', function() {
+        return this.get('node.description') !== null && this.get('node.description') !== '';
+    }),
+    // Does preprint have saved subjects?
+    savedSubjects: Ember.computed('model.subjects.@each', function() {
+        return this.get('model.subjects').length !== 0;
+    }),
+    // Preprint can be published once all required sections have been saved.
+    allSectionsValid: Ember.computed('savedTitle', 'savedFile', 'savedAbstract', 'savedSubjects', 'authorsValid', function() {
+        return this.get('savedTitle') && this.get('savedFile') && this.get('savedAbstract') && this.get('savedSubjects') && this.get('authorsValid');
     }),
-
     ////////////////////////////////////////////////////
-    // Fields used in the "basics" section of the form.
-    // Proxy for "basics" section, to support autosave when fields change (created when model selected)
-    basicsModel: Ember.computed.alias('node'),
-    basicsTitle: Ember.computed('node', function() {
-        var node = this.get('node');
-        return node ? node.get('title') : null;
+    // Fields used in the "upload" section of the form.
+    ////////////////////////////////////////////////////
+    preprintFileChanged: Ember.computed('model.primaryFile', 'selectedFile', 'file', function() {
+        // Does the pending primaryFile differ from the primary file already saved?
+        return this.get('model.primaryFile.id') !== this.get('selectedFile.id') || this.get('file') !== null;
+    }),
+    titleChanged: Ember.computed('node.title', 'nodeTitle', function() {
+        // Does the pending title differ from the title already saved?
+        return this.get('node.title') !== this.get('nodeTitle');
+    }),
+    uploadChanged: Ember.computed('preprintFileChanged', 'titleChanged', function() {
+        // Are there any unsaved changes in the upload section?
+        return this.get('preprintFileChanged') || this.get('titleChanged');
     }),
-    basicsAbstract:  Ember.computed('node', function() {
+    ////////////////////////////////////////////////////
+    // Fields used in the "basics" section of the form.
+    ////////////////////////////////////////////////////
+    basicsAbstract:  Ember.computed('node.description', function() {
+        // Pending abstract
         var node = this.get('node');
         return node ? node.get('description') : null;
     }),
+    abstractChanged: Ember.computed('basicsAbstract', 'node.description', function() {
+        // Does the pending abstract differ from the saved abstract in the db?
+        var basicsAbstract = this.get('basicsAbstract');
+        var nodeDescription = this.get('node.description');
+        var changed = false;
+        if (basicsAbstract !== null) {
+            changed = basicsAbstract.trim() !== nodeDescription;
+        }
+        return changed;
```

Comment:
slight tweak with basicsAbstract !== null, so changing an abstract to an empty string will register as a change.